packages/core/src/checkout-buttons/strategies/braintree/braintree-venmo-button-strategy.ts

```diff
@@ -131,7 +138,25 @@ export default class BraintreeVenmoButtonStrategy implements CheckoutButtonStrat
         });
     }
 
-    private async _handlePostForm(payload: BraintreeTokenizePayload): Promise<void> {
+    private async _getBuyNowCartId(buyNowInitializeOptions?: BuyNowInitializeOptions) {
```

Comment:
It might be better to rename the method to `createBuyNowCart` and return a `buyNowCart`